### ISSUE TITLE
[Bug] Mailer_1_0 not working

### ISSUE THREAD

**Issue by SOCOST** [TheHive-Project/Cortex-Analyzers#835]
**Describe the bug**
Last version of the responder "Mailer" is showing error "description is missing", although the parameter is filled.

**To Reproduce**
1. Create a case and run the responder "Mailer_1_0"

**Expected behavior**
Send an email to the mail account especified in the tag "mail:"

**Complementary information**
This is the report from Cortex:
  "errorMessage": "description is missing",
  "input": "{\"data\":{\"severity\":2,\"owner\":\"tecnico01\",\"_routing\":\"AXPXbxuYQ0lZu2cicNpz\",\"flag\":false,\"updatedBy\":\"carloss\",\"customFields\":{},\"_type\":\"case\",\"description\":\"Test\",\"title\":\"Test Mailer\",\"tags\":[\"Test\",\"mail: XXXXX\"],\"createdAt\":1597047249654,\"_parent\":null,\"createdBy\":\"carloss\",\"caseId\":27,\"tlp\":2,\"metrics\":{},\"_id\":\"AXPXbxuYQ0lZu2cicNpz\",\"id\":\"AXPXbxuYQ0lZu2cicNpz\",\"_version\":2,\"pap\":2,\"startDate\":1597046520000,\"updatedAt\":1597047259138,\"status\":\"Open\"},\"dataType\":\"thehive:case\",\"tlp\":2,\"message\":\"\",\"parameters\":{\"user\":\"carloss\"},\"config\":{\"proxy_https\":null,\"cacerts\":null,\"smtp_pwd\":\"XXXXX\",\"check_tlp\":true,\"max_tlp\":2,\"smtp_host\":\"smtp.office365.com\",\"check_pap\":true,\"max_pap\":2,\"smtp_port\":587,\"jobTimeout\":30,\"smtp_user\":\"XXXXX\",\"proxy_http\":null,\"from\":\"XXXXX\"},\"pap\":2}",
  "success": false

**Work environment**
- Client OS: Windows 10 v1909
- Server OS: Ubuntu 20.04 LTS
 - Browse type and version: Chrome Versión 84.0.4147.105 (Build oficial) (64 bits)
- Cortex version: 3.0.1-1
 - Cortex Analyzer/Responder name: Mailer 1.0
- Cortex Analyzer/Responder version: 1.0

![image](https://user-images.githubusercontent.com/68221026/89764430-a4cd1300-daf4-11ea-8b7f-586c1889c77b.png)

Thank you

**Comment by garanews** [comment]
fixed in 2.9.0

**Comment by priamai** [comment]
Please re-open I am with Cortex  3.1.1-1 and same exact bug but for alerts.
Checking the code:

```
        elif self.data_type == "thehive:alert":
            description = self.get_param(
                "data.case.description", None, "description is missing"
            )
```

and my debug log suggests that for alerts it should be instead:

```
        elif self.data_type == "thehive:alert":
            description = self.get_param(
                "data.description", None, "description is missing"
            )
```

Please let me know if you need more info.
